docs/reference/modules/admin-bar.md

```diff
@@ -0,0 +1,125 @@
+For example
+```
+deployment
+lib
+modules
+┗ @apostrophecms
+┆              ┗ admin-bar
+┆                        ┗ i18n
+┆                        ┆    ┗ en.json
+┆                        ┆      fr.json
+┆                          index.js 👈
+@node_modules
+app.js
+```
+
+The options passed in this manner will configure the existing `node_modules->apostrophe/lib/modules/admin-bar` module options through implicit subclassing. This same type of configuration takes place when you create a `modules->@apostrophecms->pages->index.js` file in the project to add configuration to the main `page` module.
+
+|  Property | Type | Description |
+|---|---|---|
+|`groups` | Array | Adds one or more menu item group objects to be displayed in dropdown menus |
+
+### `groups`
+
+The `groups` option takes an array of one or more objects that group several menu items together in the admin bar as a dropdown menu. Each of the `groups` objects requires a `label` and an array of menu `items`. The `label` will be used as the label displayed in the menu. The `items` array contains the names of the individual menu items you want to appear in the dropdown, entered in the order you want them to appear. Note: Menu names for `piece-type` items are the name of the piece-type, not the label. For core items, like 'Images', the name is prefixed - `@apostrophecms/image`.
+
+**Example**
+```javascript
+// modules/@apostrophecms/admin-bar/index.js
+module.exports = {
+ options: {
+   groups: [
+     {
+       name: 'media',
+       label: 'Media',
+       items: [
+         '@apostrophecms/image',
+         '@apostrophecms/file',
+         '@apostrophecms/image-tag',
+         '@apostrophecms/file-tag'
+       ]
+     }
+   ]
+ }
+};
+```
+This will result in grouping those four core modules into a single dropdown menu displayed as 'Media' on the menu bar.
+![ApostropheCMS admin bar with open dropdown menu titled 'Media'](/images/group-menu.png)
+
+## Featured methods
+
+The following method belongs to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/admin-bar/index.js) for all methods that belong to this module.
+<!-- Some are used within the module and would just create noise here. -->
+
+Because this module has an alias, you can call these from another module from the alias path. For example, `self.apos.adminBar.add()`.
+
+### `add(name, label, permission, options)`
+
+Add an item to the menu bar.
+
+The `name` for the item must be unique within the menu bar to avoid conflicts. When the menu item is clicked, the `name` argument will be emitted on `apos.bus` as the value of an `admin-menu-click` event. If this item controls a specific modal, this will be caught by `TheAposModals` to display the correct modal. So, `name` should be the module name with a `:editor` or `:manager` suffix. For example, `@apostrophecms/global:editor`.
+
+**Example**
```

Comment:
This is front end code. Everywhere you're presenting front end code it's vital to be crystal clear it is front end code and where you would have to put it in order for it to work, with a reference to https://v3.docs.apostrophecms.org/guide/custom-ui.html to make sure users read first about how extending the admin UI works before delving into that.

Reply:
Changed.